packages/cli/test/unit/ConcurrencyQueue.test.ts

```diff
@@ -0,0 +1,58 @@
+import { ConcurrencyQueue } from '@/ConcurrencyQueue';
+
+beforeAll(() => {
+	jest.useFakeTimers();
+});
+
+afterAll(() => {
+	jest.useRealTimers();
+});
```

Comment:
Is this needed? If we don't do this, fake timers will bleed over to other suites?